ARExperiment/ARExperiment/ARExampleViewController.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+class ARExampleViewController: UIViewController {
+
+    func setbackgroundColor() {
+        self.view.backgroundColor = .white
+    }
+
+    func setNavigationBar(controllerName: String) {
```

Comment:
can you have a think about these methods? particularly naming and division of responsibilities